.github/workflows/ci.yml

```diff
@@ -54,6 +54,11 @@ jobs:
       cedar_policy_ref: refs/heads/${{ needs.get-branch-name.outputs.branch_name }}
       cedar_examples_ref: ${{ github.href }}
 
+  tinytodo-go:
+    uses: ./.github/workflows/build_tiny_todo_go_reusable.yml
+    with:
+      cedar_examples_ref: ${{ github.href }}
```

Comment:
shouldn't this also need a ref of which commit of cedar-go to use?  or is cedar-go CI not configured to run cedar-examples as a downstream?

Reply:
Based on [this line](https://github.com/cedar-policy/cedar-examples/blob/230f2918d47d576cb5b727687a0df3bbaa50de55/tinytodo-go/go.mod#L6), I believe that `cedar-go` is pinned to [this commit](https://github.com/cedar-policy/cedar-go/commit/a71e93ee6ae7c5eb5c59e84c48f08d6b078d0168). Added a comment to this effect for future reference.

`cedar-go` does not run this example in their CI -- I'll open an issue.